examples/tiny-kafka/manifests/kafka.yaml

```diff
@@ -1,8 +1,3 @@
-apiVersion: v1
```

Comment:
Was this purposefully removed? There doesn't seem to be anything elsewhere that would create this namespace (the charts definition in the zarf.yaml for this example creates the kafka-operator namespace).

I'll be adding it back in a bit but wanted to make sure I wasn't missing anything.

Reply:
After looking at this more I think there's an issue here that needs a little tinkering still.  The expected `kafka-demo` namespace isn't being created and the helm install/upgrade is erroring out because of that. Seems like other manifests aren't auto-created by helm and we'll need to do that if we encounter a manifested not already defined.

Reply:
@jeff-mccoy

Reply:
please let me know how this goes tomorrow, that was actually a really nasty problem in the end, 04a801d4160ceb0ae153586cc6db73e0379455f6 should correct this behavior

Reply:
Seems good to me now, thanks.

Reply:
This sounds worthy of a robust couple of tests.

Reply:
Yeah I think so, we're already testing no namespace defined in basically every example.  A test with a package that creates and upgrades a package with a namespace defined in the manifests would probably be good